### DISCUSSION TITLE
UDP Port conflict

### DISCUSSION THREAD

**Discussion by vik748** [gitpython-developers/GitPython#1939] [Q&A] [answered]
We have a weird situation where our UDP functionality (ports 11011 or 11013) breaks after importing gitpython into the code.  I couldn't find any information on the ports being used by gitpython, can someone provide information on whether gitpython open TCP / UDP ports and if so which ones.

**Comment by Byron** [comment, marked as answer]
I'd be unaware of GitPython doing anything with networking at all, or mess with global settings.

**Reply by vik748** [reply]
Thanks for the response, the root cause turned out to be something else on our end.
